### PR TITLE
migrate to JUnit from TestNG

### PR DESCRIPTION
**Pull request by pawelsiwon**
PR for #435

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -272,18 +272,18 @@
             </dependency>
             <dependency>
                 <groupId>org.mockito</groupId>
-                <artifactId>mockito-testng</artifactId>
-                <version>0.4.31</version>
+                <artifactId>mockito-junit-jupiter</artifactId>
+                <version>5.0.0</version>
```
Comment (slawekjaranowski) [inline pom.xml:276]: We need `4.11.0`, `5.x` require jdk 11 at least

Comment (pawelsiwon) [reply pom.xml]: Updated

**Comment by slawekjaranowski** [comment]
@pawelsiwon please look at build result ... the same should be locally 😄

**Comment by pawelsiwon** [comment]
Sorry for the errors, I am working on machine that was freshly configured for development (first time using MacOS) and I thought it's related to my environment. I've updated, should I squash before approval?

**Review by slawekjaranowski — APPROVED** [review]

**Pull request merged**